x-pack/plugins/infra/public/alerting/common/components/alert_preview.tsx

```diff
@@ -271,33 +263,27 @@ export const AlertPreview: React.FC<Props> = (props) => {
                 iconType="alert"
               >
                 {previewError.body && (
-                  <FormattedMessage
-                    id="xpack.infra.metrics.alertFlyout.alertPreviewErrorDesc"
-                    defaultMessage="Try again later, or {viewTheError}."
-                    values={{
-                      viewTheError: <EuiLink onClick={onOpenModal}>view the error</EuiLink>,
-                    }}
-                  />
+                  <>
+                    <FormattedMessage
+                      id="xpack.infra.metrics.alertFlyout.alertPreviewErrorDesc"
+                      defaultMessage="Please try again later or see details for more information."
+                    />
+                    <EuiSpacer size={'s'} />
+                    <EuiAccordion
+                      id="alertErrorDetailsAccordion"
+                      buttonContent={i18n.translate(
```

Comment:
Change `buttonContent` so that it uses <EuiText size="s"> so that it matches the font size of other text in the callout